SU2_CFD/src/solvers/CMeshSolver.cpp

```diff
@@ -678,6 +739,55 @@ void CMeshSolver::SetBoundaryDisplacements(CGeometry *geometry, CNumerics *numer
     }
   }
 
+
+  /*--- Match deform plane is not clamped ---*/
+  for (iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++) {
+    if (config->GetMarker_All_Match_Deform_Mesh(iMarker) == YES) {
+      const su2double* nodeCoord[MAXNNODE_2D] = {nullptr};
+
```

Comment:
On the subject of duplication, the FEASolver already implements a symmetry BC, it should be possible to use that instead of duplicating its code.